lib/teiserver/coordinator/consul_server.ex

```diff
@@ -827,7 +827,9 @@ defmodule Teiserver.Coordinator.ConsulServer do
     player_list = list_players(state)
 
     {player_rating, player_uncertainty} =
-      BalanceLib.get_user_rating_value_uncertainty_pair(user.id, "Team")
+      # TODO FIX THIS BUG
```

Comment:
I believe you can check state.host_teamcount and state.host_teamsize

Reply:
Thanks, I will try that.
Modifying it a bit to include Team FFA (team_count > 2, team_size > 1)

For reference this is already reported in issue https://github.com/beyond-all-reason/teiserver/issues/85

Reply:
Be aware that there is some issue with Team FFA. Search the code base for this
```
# Temporary solution until Team FFA ratings are fixed
```

I haven't looked too deeply into it.

Reply:
Yea I am aware.
Team FFA is not used because it's played very rarely, it is probably the least played rating type, so there are just not enough matches for the ratings to adjust and create good balance. 
```ex
if player_count > team_count, do: "Team", else: "FFA"
```
For that reason TeamFFA game type is actually using Team rating type when balancing because those ratings are much more accurate even if not specifically for Team FFA mode. 
TeamFFA still gets updated based on the outcome of TeamFFA matches, it's just currently unused for balancing,